### DISCUSSION TITLE
Windows ARM64 wheel

### DISCUSSION THREAD

**Discussion by infinitepower18** [python-pillow/Pillow#6248] [General]
Python 3.11 will be released later this year and will include native support for Windows ARM64, do you have plans to publish a Windows ARM64 wheel?

**Comment by radarhere** [comment]
At the moment, our Windows wheels are kindly generated by @cgohlke for each release. So that is a question for him.

**Comment by cgohlke** [comment]
Don't know yet. In principle I would be interested. Practically, I do not have access to ARM64 hardware for building, testing, and using the binaries. Cross compiling on AMD64 should be possible. It may require some patching of libraries and build tools...
